src/messages/responses/create-signing-key/create-signing-key.ts

```diff
@@ -34,3 +37,27 @@ export class CreateSigningKeyResponse {
     return this.expiresAt;
   }
 }
+
+export class Error extends CreateSigningKeyResponse {
+  private readonly _innerException: SdkError;
+  constructor(err: SdkError) {
+    super();
+    this._innerException = err;
+  }
+
+  public message(): string {
+    return this._innerException.wrappedErrorMessage();
+  }
+
+  public innerException(): object {
+    return this._innerException;
+  }
+
+  public errorCode(): MomentoErrorCode {
+    return this._innerException.errorCode;
+  }
+
+  public override toString(): string {
+    return super.toString() + ': ' + this.message();
+  }
+}
```

Comment:
This appears to be a duplicate of CreateCacheResponse's Error. Can this be DRYed up with a [mixin](https://www.typescriptlang.org/docs/handbook/mixins.html)?

Reply:
I'll add an issue to look at this as a followup. I looked into mixins for this but spent too much time wrestling with it to justify adding it in this PR.